main.py

```diff
@@ -1,5 +1,6 @@
 import math
 #comment example
+#new test comment
 print("""Мультипарадигмальні мови програмування, Lab№1
```

Comment:
never use cyrilic in your code. If you cannot write in english, type a transliteration.